### PR TITLE
Implement preset

### PR DESCRIPTION
**Pull request by novemberborn**
For https://github.com/avajs/ava/issues/1193.

### CODE REVIEW

**Review by jfmengels — COMMENTED** [review]
Isn't stage 4 just basically ES2017? (like https://babeljs.io/docs/plugins/preset-es2017) Sure, the spec has not been finalized IIRC, but they are pretty much the same thing. The plugin name is not the most apt name IMO. `babel-preset-node-es2017` maybe?

I can't remember any additions in support in v7 compared to Node v6, are there any? Otherwise it might be interesting to have an additional config.

**Comment by novemberborn** [comment]
> Isn't stage 4 just basically ES2017?

Right now, yes. The next proposal to reach stage 4 is likely to be part of ES2018. Naming this plugin `es2017` just makes it out of date later this year.

> I can't remember any additions in support in v7 compared to Node v6, are there any? Otherwise it might be interesting to have an additional config.

The exponentiation operator is one of them. Node.js 7 has a limited lifetime though, so I didn't want to specifically cater to it.

**Comment by sindresorhus** [comment]
Looks good. :shipit: when Travis is happy.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  ],
+  "author": "Mark Wubben (https://novemberborn.net/)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/avajs/babel-preset-stage-4/issues"
+  },
+  "homepage": "https://github.com/avajs/babel-preset-stage-4#readme",
+  "devDependencies": {
+    "ava": "^0.17.0",
+    "coveralls": "^2.11.15",
```
Comment (sindresorhus) [inline package.json:35]: Adding coverage and coveralls for this tiny module feels a bit overkill, but I guess it's ok, since you've already done the work.

**Comment by jfmengels** [comment]
> Right now, yes. The next proposal to reach stage 4 is likely to be part of ES2018.

Right. I didn't remember if it was finalized, apparently it is. :+1: 

> Naming this plugin es2017 just makes it out of date later this year.

Well, that depends on what you want to do. If we wish to enable every feature that has and will reach stage 4, then yes. If that's we want, :+1: with me.

> The exponentiation operator is one of them. Node.js 7 has a limited lifetime though, so I didn't want to specifically cater to it.

:+1:

File: package.json (hunk 1 above)
Comment (novemberborn) [reply package.json:35]: Just trying to be fancy 😜

**Comment by novemberborn** [comment]
> If we wish to enable every feature that has and will reach stage 4, then yes. If that's we want, 👍 with me.

We do.

**Review by sindresorhus — APPROVED** [review]

**Pull request merged**